src/main/java/net/snowflake/client/jdbc/SnowflakeResultSetSerializableV1.java

```diff
@@ -155,6 +155,10 @@ public String toString() {
   int resultSetType;
   int resultSetConcurrency;
   int resultSetHoldability;
+  boolean treatNTZAsUTC;
+  boolean formatDateWithTimezone;
+  boolean useSessionTimezone;
+  boolean getDateUseNullTimezone;
```

Comment:
Whether the name could be different than starting with get...? Maybe should....? wdyt?

Reply:
JDBC_GET_DATE_USE_NULL_TIMEZONE is the name of the property that influences this field. I don't like this name, but I think it should be as similar to original property.